Columbus/frontend/src/pages/Profile/Profile.js

```diff
@@ -117,137 +76,6 @@ function Profile(props) {
-                          textAlign: "center",
-                        }}
-                      >
-                        <Avatar
-                          alt="r b"
-                          src=""
-                          elevation={10}
-                          style={{
-                            width: 40,
-                            height: 40,
-
-                            marginRight: "4%",
-                          }}
-                        >
-                          {item.substring(0, 2).toUpperCase()}
-                        </Avatar>
-                        <h3 style={{ margin: 0, marginTop: "3%" }}>{item}</h3>
-                      </Container>
-                    );
-                  })}
-                  {/* {getFollowing()} */}
-                </Dialog>
-                <p
-                  style={{ margin: 20, fontSize: 30 }}
-                  onClick={handleClickOpenDialogFollower}
-                >
-                  Followers: {nofFollowers}
-                </p>
-                <Dialog
-                  open={OpenDialogFollower}
-                  onClose={handleCloseDialogFollower}
-                  style={{ minWidth: 300, borderRadius: 10 }}
-                >
-                  <DialogTitle
-                    id="simple-dialog-title"
-                    style={{ minWidth: 300, textAlign: "center" }}
-                  >
-                    Followings
-                  </DialogTitle>
-                  {follower.map((item, index) => {
-                    return (
-                      <Container
-                        key={index}
-                        style={{
-                          display: "flex",
-                          flexDirection: "row",
-                          padding: "4%",
-                          textAlign: "center",
-                        }}
-                      >
-                        <Avatar
-                          alt="r b"
-                          src=""
-                          elevation={10}
-                          style={{
-                            width: 40,
-                            height: 40,
-
-                            marginRight: "4%",
-                          }}
-                        >
-                          {item.substring(0, 2).toUpperCase()}
-                        </Avatar>
-                        <h3 style={{ margin: 0, marginTop: "3%" }}>{item}</h3>
-                      </Container>
-                    );
-                  })}
-                  {/* {getFollowing()} */}
-                </Dialog>
-              </Container>
-            </div>
-          </Container>
-          <CreatePostDialog></CreatePostDialog>
-          {postIds.length === 0 && renderEmptyPost()}
-          {postIds.length !== 0 &&
-            postIds.map((item, index) => {
-              console.log("item: ", item.id);
-              return <PostCard key={index} props={{ id: 3 }}></PostCard>;
-            })}
```

Comment:
Should this be deleted? Isn't this the whole profile page?

Reply:
This was the practice app’s profile page and it was giving sone import errors. So I cleaned up the Profile page in this branch. We will merge the new version in Cf-9

Reply:
This might create conflicts. But it look goods.